R/derive_param_response.R

```diff
@@ -210,13 +219,16 @@ derive_param_response <- function(dataset,
 
   # ---- Select the 1st response and add a new PARAMCD to the input dataset ----
   dataset %>%
-    derive_param_first_event(
+    derive_param_first_eventl(
```

Comment:
derive_param_first_eventl, is the l at the end expected?

Reply:
This was a typo.. I have updated.